modules/prebidServerBidAdapter/index.js

```diff
@@ -676,6 +676,27 @@ const OPEN_RTB_PROTOCOL = {
   interpretResponse(response, bidderRequests) {
     const bids = [];
 
+    // Should log all bidders server response times, not only if they had successful seatBid
+    const serverResponseTimeMs = utils.deepAccess(response, 'ext.responsetimemillis');
```

Comment:
Is getting data for a bidder out of a response property and adding that to the bidder request done commonly? This same logic is repeated here for response times and errors. If this might also be done for other fields, I'd suggest moving this to a separate function

Reply:
good point!

Yeah I got lazy and copy and pasted to test, told myself I would clean it up but never did obviously!

I'll make a generalized func.